### PR TITLE
DB

### PR DESCRIPTION
**Pull request by TMikna**
Firsly I wrote T-SQL communication with DB, but soon realized that it can quicly cecome complex.
So, knowing that we will need Entity Framework anyway, I tried, and finally managed to create connection using IT. Work is not complete, also might be issues with nameing, package confiduration, though it allows create connection. Now left to create a class which will use all this stuff and can add antal models and deal with dependencies

### CODE REVIEW

File: SafePlace/DB/DataContext.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+
+        /// <summary>
+        /// Configure entities
+        /// </summary>
+        /// <param name="modelBuilder"></param>
+        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        {
+            // Set default schema
+            modelBuilder.HasDefaultSchema("Topas");
```
Comment (dariuskrtn) [inline SafePlace/DB/DataContext.cs]: Is this a database name? If so, then it should be moved to config.

File: SafePlace/DB/DataContext.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                .Map(ap =>
+                {
+                    ap.MapLeftKey("Person");
+                    ap.MapRightKey("Camera");
+                });
+
+            #region Auto-generated
+            modelBuilder.Entity<Camera>()
+                .Property(e => e.IPAddress)
+                .IsUnicode(false);
```
Comment (dariuskrtn) [inline SafePlace/DB/DataContext.cs]: If this isUnicode sets db encoding then it should definitely be unicode

Comment (TMikna) [reply SafePlace/DB/DataContext.cs]: Well, this one was Genereted by EF itself, so I just left as it is

File: SafePlace/DB/DataContext.cs (hunk 1 above)
Comment (TMikna) [reply SafePlace/DB/DataContext.cs]: It's schema name, databse name already in config

**Pull request merged**